lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -1683,20 +1685,35 @@ public void realtime_presence_attach_implicit_subscribe_fail() throws AblyExcept
 		AblyRealtime ably = null;
```

Comment:
I'm a bit confused because it's clearly different here:
https://github.com/ably/docs/pull/260/commits/91c700a4f32d0a1139676cb0a7648cb899f35143